.github/dependabot.yml

```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
-  - package-ecosystem: "yarn"
+  - package-ecosystem: "npm"
```

Comment:
Huh, interesting. If I'm reading this correctly [this](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem) indicates a lock file is required as well.

For interest's sake, what does this change exactly?

Reply:
I think that `npm` should support the yarn lock file. I got a notification earlier that `yarn` was actually not supported which might explain why we haven't been getting dependabot updates for this repo.